libvast/vast/system/reader_command.hpp

```diff
@@ -63,10 +63,9 @@ class reader_command : public reader_command_base {
     Reader reader{std::move(*in)};
     auto src = self->spawn(default_source<Reader>, std::move(reader));
     // Supply an alternate schema, if requested.
-    auto schema_file = get_or(options, "schema",
-                              defaults::command::schema_path);
-    if (!schema_file.empty()) {
-      auto str = load_contents(schema_file);
+    auto schema_file = caf::get_if<std::string>(&options, "schema");
+    if (schema_file && !schema_file->empty()) {
```

Comment:
@Neverlord, Is it ever possible for an option to be an empty string?

Reply:
Sure. A `vast.ini` like this will insert an empty string:

```
[global]
schema=""
```

So, I think the check is reasonable.

Reply:
Okay, good to know.